docs/index.rst

```diff
@@ -13,6 +13,7 @@ Welcome to Tianshou!
 * :class:`~tianshou.policy.DQNPolicy` `Double DQN <https://arxiv.org/pdf/1509.06461.pdf>`_
 * :class:`~tianshou.policy.DQNPolicy` `Dueling DQN <https://arxiv.org/pdf/1511.06581.pdf>`_
 * :class:`~tianshou.policy.C51Policy` `Categorical DQN <https://arxiv.org/pdf/1707.06887.pdf>`_
+* :class:`~tianshou.policy.C51Policy` `Rainbow DQN <https://arxiv.org/pdf/1707.02298.pdf>`_
```

Comment:
```suggestion
* :class:`~tianshou.policy.RainbowPolicy` `Rainbow DQN <https://arxiv.org/pdf/1707.02298.pdf>`_
```

Do we need to create such a policy class? Or at least provide a `test_rainbow.py`.

Reply:
It seems that we can get away with reusing `C51Policy` because all the other aspects of Rainbow DQN are done in either the network or the buffer. I think this is a testimony of the great design of tianshou classes, which separates concerns nicely.

I added `test_rainbow.py` for unit testing.